cmd/kyverno/main.go

```diff
@@ -484,10 +474,10 @@ func main() {
 		os.Exit(1)
 	}
 
-	pInformer.Start(stopCh)
+	kyvernoInformer.Start(stopCh)
```

Comment:
@eddycharly are all these changes expected? The original PR just contained the changes to the cert manager file before the merge commit?

Reply:
Yes, I took the opportunity to rename a couple of variables in main too, sorry.